src/bot/MessageHandler.ts

```diff
@@ -44,5 +44,15 @@ export default class MessageHandler {
     }
 
     commandToRun.run(message, params);
+
+    if (deleteMessages && !message.deleted && !this.wasMessageAlreadyDeleted(message)) {
```

Comment:
`config` is already imported. as an example check L23 which just uses `config.prefix.length`

i think you can just skip using a parameter, and simply use `config.deleteMessages` here

Reply:
i'd also love for the command to finish first, and then delete the message, so everyone knows what's up 🤙